docs/reference/MIGraphX-dev-env-vars.rst

```diff
@@ -160,6 +160,14 @@ Model performance tunable variables change the compilation behavior of a model.
 
       | Default: Split-k performance configurations are turned off.
 
+  * - | ``MIGRAPHX_ENABLE_FLASH_DECODING``
+      | Turns on flash decoding for attention fusion and sets the number of splits along the key-value sequence dimension.
+    
+    - | ``0``: Flash decoding is turned off (i.e., number of splits is 0).
+      | ``N`` (where N > 1): Enables flash decoding with N splits along the key-value sequence dimension. For example, ``2`` enables flash decoding with 2 splits, ``4`` with 4 splits, etc.

```

Comment:
Id call this flash_decoding_num_splits or something. "Enable" makes it seem like a on/off flag only